### DISCUSSION TITLE
SRS Document

### DISCUSSION THREAD

**Discussion by Hopercsd** [twbs/bootstrap#31691] [General] [answered]
Hi, I am very new to open source programming and I want to try to write a Software Requirements Specification document(SRS) about boostrap. Is there maybe already one?

**Comment by techboyg5** [comment]
Bootstrap is not software. It's a CSS framework and nothing else.

**Reply by mdo** [reply]
This isn't accurate—yes, it's software, and no, it's not just CSS.

**Comment by mdo** [comment, marked as answer]
We don't currently have one, and I'm unsure that we'd even need one.
